### PR TITLE
fix(tm2): Enable coin benchmark tests after fixing panic error

### PR DESCRIPTION
**Pull request by sunspirit99**
Relate to https://github.com/gnolang/gno/issues/907

Make a few small fixes to get this test working

<details><summary>Contributors' checklist...</summary>

- [ ] Added new tests, or not needed, or not feasible
- [ ] Provided an example (e.g. screenshot) to aid review or the PR is self-explanatory
- [ ] Updated the official documentation or not needed
- [ ] No breaking changes were made, or a `BREAKING CHANGE: xxx` message was included in the description
- [ ] Added references to related issues and PRs
- [ ] Provided any useful hints for running manual tests
- [ ] Added new benchmarks to [generated graphs](https://gnoland.github.io/benchmarks), if any. More info [here](https://github.com/gnolang/gno/blob/master/.benchmarks/README.md).
</details>

### CODE REVIEW

**Comment by notJoon** [comment]
Could you please provide more detailed context for this PR's changes?

File: tm2/pkg/std/coin_benchmark_test.go [hunk 1]
```diff
@@ -2,24 +2,30 @@ package std
...
 			b.Helper()
 
 			coinsA := Coins(make([]Coin, numCoinsA))
 			coinsB := Coins(make([]Coin, numCoinsB))
 
+			maxCoins := numCoinsA
+			if numCoinsB > numCoinsA {
+				maxCoins = numCoinsB
+			}
+			denomLength := len(fmt.Sprint(maxCoins))
```
Comment (thehowl) [inline tm2/pkg/std/coin_benchmark_test.go:20]: can use built-in `max` function

**Review by thehowl — APPROVED** [review]

**Comment by jefft0** [comment]
Removed the `review/triage-pending` label because core dev thehowl approved this PR.

File: tm2/pkg/std/coin_benchmark_test.go (hunk 1 above)
Comment (sunspirit99) [reply tm2/pkg/std/coin_benchmark_test.go]: ef2bdb5b6df7ee63140626fa78b4923101780aa9

**Pull request merged**